packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -148,6 +148,33 @@ void main() {
       expect(pickers.any((CupertinoPicker picker) => picker.backgroundColor != CupertinoColors.black), false);
     });
 
+    testWidgets('item extent default value', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        CupertinoApp(
+          home: CupertinoTimerPicker(
+            onTimerDurationChanged: (_) { },
+          ),
+        ),
+      );
+
+      final Iterable<CupertinoPicker> pickers = tester.allWidgets.whereType<CupertinoPicker>();
+      expect(pickers.any((CupertinoPicker picker) => picker.itemExtent != 32), false);
+    });
+
+    testWidgets('specified item extent default value is applied', (WidgetTester tester) async {
```

Comment:
Nit:

```suggestion
    testWidgets('specified item extent value is applied', (WidgetTester tester) async {
```

Because "default value" makes me think of the value when nothing is passed.